src/component/handlers/composite-android/DraftEditorCompositionHandlerAndroid.js

```diff
@@ -46,6 +48,52 @@ const createMutationObserverIfUndefined = () => {
   }
 };
 
+const startCompositionTimeout = (editor) => {
+  cancelCompositionTimeout();
+
+  if (!getEditorState(editor).isInCompositionMode()) {
+    //editor.update(EditorState.set(getEditorState(editor), { inCompositionMode: true }));
+    editor.silentlyUpdate(EditorState.set(getEditorState(editor), { inCompositionMode: true }));
```

Comment:
Is this just for speed or is this to get around a problem? I think it's a smart change, just curious.

Reply:
Oops, I left the commented line above in! I'll fix that at the least.
Updating the editor state triggers an `onChange` which seemed to cause issues, or at least open it up to the possibility of issues.  This way we can roughly say: "Yes the state is changing, but no dear consumer you need not concern yourself with it."